src/politech_backend/urls.py

```diff
@@ -41,7 +42,9 @@
     url(r'^api/v1/subjects/$', subject_views.SubjectList.as_view()),
 
     url(r'^api/v1/rest-auth/', include('rest_auth.urls')),
-    url(r'^api/v1/rest-auth/registration/', include('rest_auth.registration.urls'))
+    url(r'^api/v1/rest-auth/registration/', include('rest_auth.registration.urls')),
+    url(r'^api/v1/oauth/', include('social_django.urls', namespace='social')),
+    url(r'^api/v1/oauth/(?P<backend>[^/]+)/$', oauth_views.login_with_access_token)
```

Comment:
It's a variable name for what backend to authorize with, e.g. Facebook and GitHub. social_django seems to be using it because I can't rename or replace or else I get the error
`backend_1  | TypeError: wrapper() missing 1 required positional argument: 'backend'`